src/core/lib/security/credentials/tls/grpc_tls_certificate_provider.h

```diff
@@ -150,6 +150,7 @@ class FileWatcherCertificateProvider final
   }
 
   UniqueTypeName type() const override;
+  int64_t refresh_interval_sec_ = 0;
```

Comment:
If we really have to expose it (indeed I don't know of any other good way to assert this), I think we should have a getter method instead of exposing the raw field.

Reply:
Swapped to an accessor - `TestOnlyGetRefreshIntervalSecond`